File: sources/sushimi-console/app/kz/sushimi/console/models/dictionaries/UnitModel.java
package kz.sushimi.console.models.dictionaries;

import java.util.Date;


/**
 * Модель для отображения единиц измерений
 * @author Demart
 *
 */
public class UnitModel {

	public long id;
	public String name;
	public String shortName;
	public String user;
	public Date createdDate;
	public Date modifiedDate;
	
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getShortName() {
		return shortName;
	}
	public void setShortName(String shortName) {
		this.shortName = shortName;
	}
	public String getUser() {
		return user;
	}
	public void setUser(String user) {
		this.user = user;
	}
	public Date getCreatedDate() {
		return createdDate;
	}
	public void setCreatedDate(Date createdDate) {
		this.createdDate = createdDate;
	}
	public Date getModifiedDate() {
		return modifiedDate;
	}
	public void setModifiedDate(Date modifiedDate) {
		this.modifiedDate = modifiedDate;
	}
	
}

File: sources/sushimi-console/bin/public/extjs/app/model/UnitModel.js
Ext.define('SushimiConsole.model.UnitModel', {
    extend: 'Ext.data.Model',
    idProperty: 'id',
    fields: [
		{ name: 'id', type: 'int'}, 
		{ name: 'name'},
		{ name: 'shortName'},
		{ name: 'user'},
		{ name: 'createdDate', type: 'date'}, 
		{ name: 'modifiedDate', type: 'date'},
	],
});